### DISCUSSION TITLE
Immediate web server crash on cross-platform application template

### DISCUSSION THREAD

**Discussion by twoltjer** [AvaloniaUI/Avalonia#7356] [Q&A] [answered]
I've attempted this on both linux and Windows 10. I followed the Jetbrains Rider setup step published in the Getting Started guide (on the website), and created a new solution using the cross-platform application template. The web project will not launch with either IIS express or the AvaloniaBlazorSample on Windows or Linux.

When it starts, it immediately shows a browser with "Powered by Avalonia," so it seems to start at least part-way. Then, a message shows at the bottom of the window saying that "An unhandled error has occurred."

The console in Rider shows the following: 

![image](https://user-images.githubusercontent.com/6701410/149432232-6964ed38-7a97-413d-808a-85023e9b8e3c.png)

I've verified the skiasharp package has restored correctly and that I have full permissions on .nuget/packages/skiasharp. 

Anyone know what is happening here, and how to fix it?
Thanks for your help!

**Comment by kekekeks** [comment, marked as answer]
`dotnet workload install wasm-tools`
